### PR TITLE
Add PR/issue note in Contribution Workflow Example

### PR DESCRIPTION
**Pull request by mik-laj**
---
Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Target Github ISSUE in description if exists
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by mik-laj** [comment]
@nullhack Can I ask for review?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -665,6 +665,9 @@ Contribution Workflow Example
 Typically, you start your first contribution by reviewing open tickets
 at `GitHub issues <https://github.com/apache/airflow/issues>`__.
 
+If you create pull-request, you don't have to create a issue first, but if you want, you can do it.
```
Comment (nullhack) [inline CONTRIBUTING.rst]:
```suggestion
If you create pull-request, you don't have to create an issue first, but if you want, you can do it. 
```

**Review by nullhack — CHANGES_REQUESTED** [review]
Seems good, will help other new contributors. Just a small change.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (mik-laj) [reply CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -665,6 +665,9 @@ Contribution Workflow Example
 Typically, you start your first contribution by reviewing open tickets
 at `GitHub issues <https://github.com/apache/airflow/issues>`__.
 
+If you create pull-request, you don't have to create a issue first, but if you want, you can do it. 
+Creating a issue will allow you to collect feedback or share plans with other people.
```
Comment (nullhack) [inline CONTRIBUTING.rst]:
```suggestion
Creating an issue will allow you to collect feedback or share plans with other people.
```

Comment (mik-laj) [reply CONTRIBUTING.rst]: Done

**Review by nullhack — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**